docs/concepts/resolution.md

```diff
@@ -258,6 +265,46 @@ If multiple overrides are provided for the same package, they must be differenti
 [markers](#platform-markers). If a package has a dependency with a marker, it is replaced
 unconditionally when using overrides — it does not matter if the marker evaluates to true or false.
 
+## Metadata overrides
+
+Metadata overrides allow overriding the metadata of a specific package. For example, to provide
```

Comment:
My only complaint with the name ("Metadata overrides") is that the primary goal isn't to override... It's to allow the user to provide static metadata so that we can skip builds. I think override implies changing the metadata in some way, but the goal here is just to enable users to pre-provide the true metadata.

Reply:
>  I think override implies changing the metadata in some way, but the goal here is just to enable users to pre-provide the true metadata.

Fair, but I think if we want to frame it that way we need to provide a way to (1) populate the metadata automatically and/or (2) validate that it matches the distribution at install time.

I wouldn't be upset about "Dependency metadata" instead if we plan to add features like (1) and (2). What do you think?

Reply:
I do want to do (1) and (2), but I probably won't do them ASAP. I don't know what the right API looks like for (1) 🤔

Reply:
I'm now leaning towards "dependency metadata".